src/main/java/frc/robot/Simulation/DriveSim.java

```diff
@@ -37,12 +39,14 @@ public class DriveSim {
 
     private AHRS gyro;
     private AHRSSim gyroSim;
-Conversions conversions;
+    Vision vision = new Vision();
+    Drive drive = new Drive(vision);
+   
     //drivetrain sim object
     
     private DifferentialDrivetrainSim drivetrainSim  = new DifferentialDrivetrainSim(
```

Comment:
We deleted the drivesim class ages ago, it would probably be a good idea to update this branch from master....